tox.ini

```diff
@@ -51,7 +51,7 @@ description = type check ourselves
 basepython = python3.7
 commands =
     python -m mypy --config-file mypy_self_check.ini -p mypy -p mypyc
-    python -m mypy --config-file mypy_self_check.ini --baseline-file .mypy/proper_plugin.json misc/proper_plugin.py
+    python -m mypy --config-file mypy_self_check.ini --legacy misc/proper_plugin.py
```

Comment:
whats `proper_plugin.json` and why was it removed

Reply:
it was a seperate baseline for a different run, I just made that run use legacy mode instead